src/main/scala/chiseltest/internal/ThreadedBackend.scala

```diff
@@ -2,13 +2,37 @@
 
 package chiseltest.internal
 
-import java.util.concurrent.{ConcurrentLinkedQueue, Semaphore}
+import chisel3.{Clock, Data, Module}
 
-import chiseltest.{Region, TemporalParadox, ThreadOrderDependentException}
-import chisel3._
+import java.util.concurrent.{ConcurrentLinkedQueue, Semaphore}
+import chiseltest.{Region, TemporalParadox, ThreadOrderDependentException, TimeoutException}
+import chiseltest.coverage.TestCoverage
+import chiseltest.simulator.SimulatorContext
+import firrtl.AnnotationSeq
+import firrtl.annotations.NoTargetAnnotation
 
 import scala.collection.mutable
 
+trait AbstractTesterThread
```

Comment:
should be private

Reply:
both of this two was public, I think cleanup can be a separate PR?